src/crimson/os/futurized_store.h

```diff
@@ -26,6 +26,37 @@ class FuturizedCollection;
 class FuturizedStore {
 
 public:
+  class OmapIterator {
+  public:
+    virtual seastar::future<int> seek_to_first() {
+      return seastar::make_ready_future<int>(0);
+    };
+    virtual seastar::future<int> upper_bound(const std::string &after) {
+      return seastar::make_ready_future<int>(0);
+    };
+    virtual seastar::future<int> lower_bound(const std::string &to) {
+      return seastar::make_ready_future<int>(0);
+    };
+    virtual bool valid() {
+      return false;
+    };
+    virtual seastar::future<int> next() {
+      return seastar::make_ready_future<int>(0);
+    };
+    virtual std::string key() {
+      return "";
+    }
+    virtual seastar::future<std::string> tail_key() {
+      return seastar::make_ready_future<std::string>();
+    };
+    virtual ceph::buffer::list value() {
+      return ceph::buffer::list();
+    }
+    virtual int status() {
+      return 0;
+    }
+  };
```

Comment:
remove trailing ";".

Reply:
um... this is the end of the definition of class OmapIterator